src/App.js

```diff
@@ -1,25 +1,55 @@
-import logo from './logo.svg';
 import './App.css';
+import foods from './foods.json'
+import FoodBox from "./FoodBox"
+import { useState } from 'react';
+import AddFoodForm from './AddFoodForm';
+import Search from './Search'
 
 function App() {
+  const [items, setItems] = useState(foods)
+  const [deletedFood, setDeletedFood] = useState(foods)
+
+  const deadFood = (name) => {
+    const suuperSearch = items.filter((bananas) => bananas.name !== name)
+    setItems(suuperSearch);
+    setDeletedFood(suuperSearch);
+  }
+
+  const newForm = (bananas) => {
+    const newFood = [bananas, ...items]
+    setItems(newFood)
```

Comment:
For your search bar to work, you will need the following steps on your code:
1-
`setDeletedFood(newFood)`
in here.